sql/driver/driver_test.go

```diff
@@ -345,6 +408,9 @@ func TestInsertSelectDelete(t *testing.T) {
 		if _, err := db.Exec(fmt.Sprintf(`INSERT INTO %s.kv (v) VALUES ('a')`, testcase.db)); !isError(err, "missing .* primary key column") {
 			t.Fatal(err)
 		}
+		if _, err := db.Exec(fmt.Sprintf(`INSERT INTO %s.kv VALUES ($1, $2)`, testcase.db), "nil", nil); err != nil {
+			t.Fatal(err)
+		}
```

Comment:
We need another test case to verify you can't insert `NULL` into a `NOT NULL` column.